templates/Procfile

```diff
@@ -1,2 +1,2 @@
-web: bundle exec unicorn -p $PORT -c ./config/unicorn.rb
+web: bundle exec rails server Puma -p ${PORT:-3000} -c ./config/puma.rb
```

Comment:
This seems to break starting the app with Foreman. While running `bundle exec rails s puma' works, passing the config file will break with `undefined method `workers'`. This is a known issue (see https://github.com/puma/puma/issues/512)  and the example Procfile in the Heroku docs uses puma directly. Of course, the downside to that solution is losing logging...

Changing the Procfile to `web: bundle exec puma -C ./config/puma.rb` does boot the app loading the config.
but will also throw a thread warning related to the connection pool (discussion around similar issues can be found in https://github.com/puma/puma/issues/614).